examples/standard/main.tf

```diff
@@ -0,0 +1,124 @@
+##############################################################################
+# Provision an OCP cluster with one extra worker pool inside a VPC
+##############################################################################
+
+module "resource_group" {
+  source = "git::https://github.com/terraform-ibm-modules/terraform-ibm-resource-group.git?ref=v1.0.5"
+  # if an existing resource group is not set (null) create a new one using prefix
+  resource_group_name          = var.resource_group == null ? "${var.prefix}-resource-group" : null
+  existing_resource_group_name = var.resource_group
+}
+
+###############################################################################
+# VPC
+###############################################################################
+
+locals {
+  public_gateway = {
+    zone-1 = true
+    zone-2 = false
+    zone-3 = false
+  }
+  addresses = {
+    zone-1 = ["10.10.10.0/24"]
+    zone-2 = ["10.20.10.0/24"]
+    zone-3 = ["10.30.10.0/24"]
+  }
+  subnets = {
+    zone-1 = [
+      {
+        acl_name = "vpc-acl"
+        name     = "zone-1"
+        cidr     = "10.10.10.0/24"
+      }
+    ],
+    zone-2 = [
+      {
+        acl_name = "vpc-acl"
+        name     = "zone-2"
+        cidr     = "10.20.10.0/24"
+      }
+    ],
+    zone-3 = [
+      {
+        acl_name = "vpc-acl"
+        name     = "zone-3"
+        cidr     = "10.30.10.0/24"
+      }
+    ]
+  }
+}
+
+module "vpc" {
+  source              = "git::https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone-vpc.git?ref=v3.0.0"
+  resource_group_id   = module.resource_group.resource_group_id
+  region              = var.region
+  prefix              = var.prefix
+  tags                = var.resource_tags
+  name                = var.vpc_name
+  address_prefixes    = local.addresses
+  subnets             = local.subnets
+  use_public_gateways = local.public_gateway
+}
+
+##############################################################################
+# KMS
+##############################################################################
+resource "ibm_resource_instance" "kms_instance" {
```

Comment:
We may want to move to using the terraform-ibm-key-protect-* modules for this to show integration between different modules of the ecosystem. Not needed in this initial PR.

Reply:
Tracked at https://github.com/terraform-ibm-modules/terraform-ibm-base-ocp-vpc/issues/32